foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/dao/DefaultPropertyDao.java

```diff
@@ -0,0 +1,60 @@
+package com.boozallen.aissemble.configuration.dao;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Configuration::Store
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+
+import com.boozallen.aissemble.configuration.store.Property;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * DefaultPropertyDao
+ * TODO: currently this is a dummy class which can be used to check whether the request is received,
```

Comment:
Q: Is this just so tests work? I thought `VaultPropertyDao` was our initial/default implementation.

Reply:
yeah.. this default property dao can be used for testing for now. Per conversation with Ryan, the vault can be the default option for now but we still need a more generic one since vault is a bit heavy for small project.

Reply:
offline discussion and will keep this class but renamed to InMemoryPropertyDao